parser-typechecker/src/Unison/SyntaxHighlights.hs

```diff
@@ -0,0 +1,69 @@
+module Unison.SyntaxHighlights where
+
+import Unison.Util.ColorText
+import Unison.Util.Pretty             ( Pretty )
+
+-- The elements of the Unison grammar, for syntax highlighting purposes
+data Element = NumericLiteral
+             | TextLiteral
+             | BooleanLiteral
+             | Blank
+             | Var
+             | Reference
+             | Constructor
+             | Request
+             | AbilityBraces
+             -- let|handle|in|where|case|of|->|if|then|else|and|or
+             | ControlKeyword 
+             -- forall|->
+             | TypeOperator
+             | BindingEquals
+             | TypeAscriptionColon
+             -- type|ability
+             | DataTypeKeyword
+             | DataTypeParams
+             | DataType
+             -- unique
+             | DataTypeModifier
+             -- `use Foo bar` is keyword, prefix, suffix
+             | UseKeyword
+             | UsePrefix
+             | UseSuffix
+             | HashQualifier
+             | DelayForceChar
+             -- ? , ` [ ] @ |
+             -- Currently not all commas in the pretty-print output are marked up as DelimiterChar - we miss
+             -- out characters emitted by Pretty.hs helpers like Pretty.commas.
+             | DelimiterChar
+             -- ! '
+             | Parenthesis
+
+defaultColors :: Element -> ColorText -> ColorText
+defaultColors = \case 
+  NumericLiteral      -> purple
```

Comment:
Just pointing out that you can say `id` for any of these to use the default body text color of the terminal.